lib/Mongo/MongoCursor.php

```diff
@@ -154,6 +155,7 @@ protected function doQuery()
     {
         $options = $this->getOptions() + $this->options;
 
+        //$this->cursor = $this->collection->find(TypeConverter::convertLegacyArrayToObject($this->query), $options);
```

Comment:
@alcaeus I tried to apply the conversion here but it breaks a lot of tests :fearful: